data-structures-stacks/count-values.js

```diff
@@ -0,0 +1,10 @@
+export default function countValues(stack) {
+  let count = 0;
+  if (stack.peek() === undefined) {
+    return 0;
+  }
```

Comment:
The `if` block here is redundant: the same return value will be sent by the `while` block.